app/_src/policies/introduction.md

```diff
@@ -1,6 +1,661 @@
+
+Consider the two example policies below:
+
+{% policy_yaml accesslog_outbound_example %}
+```yaml
+type: MeshAccessLog
+name: example-outbound
+mesh: default
+spec:
+  targetRef: # top level targetRef
+    kind: MeshSubset
+    tags:
+        app: web-frontend
+  to:
+    - targetRef: # to level targetRef
+        kind: MeshService
+        name: web-backend
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+```
+{% endpolicy_yaml accesslog_inbound_example %}
+```yaml
+type: MeshAccessLog
+name: example-inbound
+mesh: default
+spec:
+  targetRef: # top level targetRef
+    kind: MeshSubset
+    tags:
+      app: web-frontend
+  from:
+    - targetRef: # from level targetRef
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+```
+{% endpolicy_yaml %}
+
+Using `spec.targetRef`, this policy targets all proxies that have a tag `app:web-frontend`.
+It defines the scope of this policy as applying to traffic either from or to data plane proxies with the tag `app:web-frontend`.
+
+The `spec.to[].targetRef` section enables logging for any traffic going to `web-backend`.
+The `spec.from[].targetRef` section enables logging for any traffic coming from _anywhere_ in the `Mesh`.
+
+### Omitting `targetRef`
+When a `targetRef` is not present. It is semantically equivalent to: `targetRef.kind: Mesh` meaning everything inside the `Mesh`.
+
+### Applying to specific proxy types
+The top level `targetRef` field can select a specific subset of data plane proxies. The field named `proxyTypes` can restrict policies to specific types of data plane proxies:
+- `Sidecar`: Targets data plane proxies acting as sidecars to applications.
+- `Gateway`: Applies to data plane proxies operating in Gateway mode.
+- Empty list: Defaults to targeting all data plane proxies.
+
+#### Example
+
+The following policy will only apply to gateway data-planes:
+{% policy_yaml proxytypes %}
+```yaml
+type: MeshTimeout
+name: gateway-only-timeout
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: ["Gateway"]
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        idleTimeout: 10s
+```
+{% endpolicy_yaml %}
+
+### Target resources
+
+Not every policy supports `to` and `from` levels. Additionally, not every resource can
+appear at every supported level. The specified top level resource can also affect which
+resources can appear in `to` or `from`.
+
+To help users, each policy documentation includes tables indicating which `targetRef` kinds is supported at each level.
+For each type of proxy, sidecar or builtin gateway, the table indicates for each
+`targetRef` level, which kinds are supported.
+
+#### Example tables
+
+These are just examples, remember to check the docs specific to your policy!
```

Comment:
This seems bizarre to have as an _error_, maybe we need to ignore this rule. Or maybe better in a warning box? Up to you